src/fa-screenshot.sh

```diff
@@ -42,23 +38,25 @@ function usage() {
   #print pretty usage and exit
   cat <<HELP_USAGE
 
-    $0  [-s] [-t] [-d] <destination folder> [-h]
+    $0  [-s] [-t] [-d destination folder] [-f filename] [-u url] [-h] 
 
    -s  Silent mode
    -t  Use TinyPNG API instead of pngquant library
+   -f  Filename for screenshot. If not provided, defaults to datetime
    -d  Move screenshots to given folder
    -x  How far to move the safari window on the x axis (number). Default is 640
    -h  Print this usage
```

Comment:
Just need a `-u` description in the help.

Reply:
Great catch, thanks.